automations/python/workflows/write_changelog.py

```diff
@@ -8,9 +9,11 @@
 
 file_path = Path(__file__).parents[3] / f"documentation/changelogs/{app}/{tag_date}.md"
 file_path.write_text(
-    """
-    # {tag_date}
+    dedent(
+        f"""
+        # {tag_date}
 
-    {release_body}
-    """
+        {release_body}
+        """
+    )
```

Comment:
This was missing the f tag on the string and, if you look at the commit diffs, you'll see that for some reason the multi-line string did not automatically dedent. I've passed it through dedent to fix that.